topic_folders/instructor_training/email_templates_trainers.md

```diff
@@ -133,60 +133,121 @@ Best wishes,
 
-[nombre del remitente]
+¡Hola!
+
+Según el [Etherpad de demostraciones de enseñanza](https://pad.carpentries.org/teaching-demos), te inscribiste para hacer una demostración de enseñanza en línea el [fecha] a las [hora] (usa este enlace para ver tu hora local: [enlace de timeanddate.com]). Yo seré la persona que capacita y que dirigirá la sesión.
+
+Quiero que sepas que puedo asignarte cualquiera de los segmentos de la lección que preparaste para enseñar; por lo tanto, prepárate para enseñar todas las partes de la lección que elegiste. Algunas personas se preparan para enseñar solo cinco minutos de una sección en particular y, con frecuencia, deben reprogramar la demostración porque casi nunca les asignan la sección para la que se prepararon.
+
+Una lección corresponde a una sola fila de las tablas de lecciones (https://software-carpentry.org/lessons/, http://www.datacarpentry.org/lessons/ y https://librarycarpentry.org/lessons/) y a un solo repositorio en GitHub. Algunas lecciones tienen módulos complementarios, pero no es necesario que te prepares para enseñar esos módulos en tu demostración de enseñanza.
+
+Por ejemplo, si elegiste “terminal de Unix”, te puedo asignar cualquier sección que figure en http://swcarpentry.github.io/shell-novice/.
+
+Recuerda que esto no es una prueba determinante, sino una oportunidad favorable para dar y recibir comentarios sobre un estilo de presentación más pulido. Si te pido que repitas tu demostración, te enviaré por correo electrónico información con instrucciones claras sobre lo que se debe cambiar después de la demostración. Durante la demostración, recibirás públicamente solo comentarios cualitativos, no se te informará si este paso de finalización se considera completado.
+
+Espero verte pronto.
+
+[Nombre de remitente]
+
+#### Invitación para aprendices para que repitan la demostración de enseñanza
+
+**Nota para las personas que capacitan:** Incluso con una plantilla, puede ser difícil redactar y enviar estos correos electrónicos. Si no están seguras, no duden en comunicarse con la comunidad de personas que capacitan o con el Equipo central para que las ayuden a decidir, personalizar o responder preguntas sobre estos mensajes.
+
+**Asunto:** Capacitación de Instructoras de The Carpentries: demostración de enseñanza
+
+Hola, [nombre de aprendiz]:
+
+Gracias por hacer una demostración de enseñanza. Aunque demostraste [mencionar algo que haya hecho bien, p. ej., “buen dominio del material de referencia”], te invito a que vuelvas a hacer la demonstración de enseñanza. El motivo principal es que [motivo]. [Explicación de lo que te gustaría ver esta vez y por qué es importante para las prácticas de enseñanza de The Carpentries].
+
+Nos complace tenerte como instructora de The Carpentries y queremos que te sumes a nuestro equipo. Sé que es complicado destinar tiempo para estas sesiones, pero espero que la oportunidad de seguir practicando y de recibir más comentarios sea beneficiosa para ti.
```

Comment:
```suggestion
```
The paragraph says that we are happy to welcome the person as an Instructor. I believe this is a mistake also present in the English version.

Reply:
This may be one of those things that does not translate well. In the English version, the purpose is to reassure the trainee that we DO want them to become an Instructor, even though we are asking them to repeat their demo. Maybe this reassurance could be provided in a different way; otherwise this could be removed for now.